docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md

```diff
@@ -0,0 +1,347 @@
+---
+description: Provides guidance on how to configure the auto-update and repair settings of a Windows App.
+external help file: Microsoft.Windows.Appx.PackageManager.Commands.dll-help.xml
+Module Name: Appx
+ms.date: 06/07/2021
+online version: https://docs.microsoft.com/powershell/module/appx/reset-appxpackage?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Set-AppxPackageAutoUpdateSettings
+---
+
+# Set-AppxPackageAutoUpdateSettings
+
+## SYNOPSIS
+Available in the Windows Insider Preview Builds of windows 10, is the `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet. The `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides access to configure a specific Windows App's Auto Update and Repair settings. Including pausing update checks.
```

Comment:
```suggestion
Available in the Windows Insider Preview Builds of windows 10, is the `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet. The `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides access to configure a specific Windows App's Auto Update and Repair settings. Including pausing update checks.
```
```suggestion
The `Get-AppxPackageAutoUpdateSettings` PowerShell cmdlet is available in the Windows Insider Preview Builds of Windows 10. The `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides access to configure a specific Windows App's Auto Update and Repair settings. Including pausing update checks.
```

Reply:
Should this comment of it being available only in the Insider Preview not be included?

Reply:
I have changed my suggestion slightly.